modules/sql-engine-api/src/main/java/org/apache/ignite/internal/sql/engine/api/kill/KillHandlerRegistry.java

```diff
@@ -28,4 +28,12 @@ public interface KillHandlerRegistry {
      * @param handler Handler to register.
      */
     void register(OperationKillHandler handler);
+
+    /**
+     * Returns operation kill handler for specified operation type.
+     *
+     * @param type operation type.
```

Comment:
```suggestion
     * @param type Operation type.
```

Also remove `@SuppressWarnings("InterfaceMayBeAnnotatedFunctional")` class annotation